### PR TITLE
Add ERC: Wrapping of bubbled up reverts

### PR DESCRIPTION
**Pull request by gretzke**
When opening a pull request to submit a new EIP, please use the suggested template: https://github.com/ethereum/EIPs/blob/master/eip-template.md

We have a GitHub bot that automatically merges some PRs. It will merge yours immediately if certain criteria are met:

 - The PR edits only existing draft PRs.
 - The build passes.
 - Your GitHub username or email address is listed in the 'author' header of all affected PRs, inside <triangular brackets>.
 - If matching on email address, the email address is the one publicly listed on your GitHub profile.

### CODE REVIEW

File: ERCS/erc-draft_wrap_revert.md [hunk 1]
```diff
@@ -0,0 +1,142 @@
+---
```
Comment (abcoathup) [inline ERCS/erc-draft_wrap_revert.md:1]:
```suggestion
---
eip: 7751
```
Assigning next sequential EIP/ERC/RIP number.

Please also update the filename.

File: ERCS/erc-draft_wrap_revert.md [hunk 2]
```diff
@@ -0,0 +1,142 @@
...
+title: Wrapping of bubbled up reverts
+description: Handling bubbled up reverts using custom errors with additional context
+author: Daniel Gretzke (@gretzke), Sara Reynolds (@snreynolds), Alice Henshaw (@hensha256), Marko Veniger (marko.veniger@tenderly.co), Hadrien Croubois (@Amxx)
+discussions-to: https://ethereum-magicians.org/t/tbd
```
Comment (abcoathup) [inline ERCS/erc-draft_wrap_revert.md:5]: Please create the discussions topic in Eth Magicians.  It just needs to have a link to this PR.

Comment (gretzke) [reply ERCS/erc-draft_wrap_revert.md:5]: I'm trying haha, but I don't see a button to create a new topic. I reached out on telegram to fix the issue

Comment (gretzke) [reply ERCS/erc-draft_wrap_revert.md:5]: image

this is how it's supposed to look like and looks like for a colleague

Comment (gretzke) [reply ERCS/erc-draft_wrap_revert.md:5]: this is how it looks like for me
image

Comment (gretzke) [reply ERCS/erc-draft_wrap_revert.md]: my colleague now posted on my behalf, I updated the link in the ERC

File: ERCS/erc-7751.md [hunk 3]
```diff
@@ -0,0 +1,143 @@
+---
+eip: 7751
+title: Wrapping of bubbled up reverts
+description: Handling bubbled up reverts using custom errors with additional context
+author: Daniel Gretzke (@gretzke), Sara Reynolds (@snreynolds), Alice Henshaw (@hensha256), Marko Veniger <marko.veniger@tenderly.co>, Hadrien Croubois (@Amxx)
+discussions-to: https://ethereum-magicians.org/t/wrapping-of-bubbled-up-reverts/20740
```
Comment (abcoathup) [inline ERCS/erc-7751.md:6]:
```suggestion
discussions-to: https://ethereum-magicians.org/t/erc-7751-wrapping-of-bubbled-up-reverts/20740
```
Added ERC number to topic title in Eth Magicians.

Comment (gretzke) [reply ERCS/erc-7751.md]: done

**Review by SamWilsn — APPROVED** [review]

**Pull request merged**